actions/helm-deploy/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM google/cloud-sdk:slim
+
+RUN apt-get install -y kubectl
+
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/v3.5.3/scripts/get-helm-3 && \
+    chmod 700 get_helm.sh && \
+    ./get_helm.sh
```

Comment:
Here is the suggestion I had which seems to work:

```suggestion
RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/v3.5.3/scripts/get-helm-3 && \
    chmod 700 get_helm.sh && \
    ./get_helm.sh && \
    helm plugin install https://github.com/thomastaylor312/helm-namespace && \
    helm repo add datarepo-helm https://broadinstitute.github.io/datarepo-helm && \
    helm repo update
```

Reply:
Did you just build and run the image locally to test it out? When I just had `helm plugin install https://github.com/thomastaylor312/helm-namespace` previously, I could run `helm namespace` if I ran this container locally, but then it didn't work in the action 😞

Reply:
I was able to test it locally and it seemed to work. Strange that it wouldn't work in the action.

Reply:
Hmm...I'll apply your suggestion so we can test it in the action too

Reply:
@fboulnois the action passed 💯 https://github.com/DataBiosphere/jade-data-repo/runs/2184745302?check_suite_focus=true